<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPrefs</class>
 <widget class="QWidget" name="AudioPrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input Channel Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Left:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_leftInputChannel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Right:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_rightInputChannel"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_leaveInputDisconnected">
        <property name="text">
         <string>Leave disconnected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output Channel Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Left:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_leftOutputChannel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Right:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_rightOutputChannel"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_leaveOutputDisconnected">
        <property name="text">
         <string>Leave disconnected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
